examples/bpk-component-ticket/examples.js

```diff
@@ -187,8 +176,6 @@ const MixedExample = () => (
     <DefaultExample />
     <VerticalExample />
     <WithoutPaddingExample />
-    <WithoutNotchesExample />
-    <WithoutNotchesVerticalExample />
     <WithHrefExample />
```

Comment:
The new example `WithoutPaddingImageExample` should include in the MixedExample, so it could be displayed on the visual test snapshot. (You need to run the script to update it)